pulp_container/app/viewsets.py

```diff
@@ -545,6 +545,33 @@ class ContainerRepositoryVersionViewSet(RepositoryVersionViewSet):
     """
 
     parent_viewset = ContainerRepositoryViewSet
+    permission_classes = (access_policy.RepositoryVersionAccessPolicyFromDB,)
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repoversion_repo_model_or_obj_perms:"
+                    "container.view_containerrepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition":
+                    "(has_repoversion_repo_model_or_obj_perms:container.delete_containerrepository "
+                    "or has_repoversion_repo_model_or_obj_perms:"
+                        "container.modify_content_containerrepository)",
```

Comment:
Any formatting suggestions?
black does this:
```
                "condition":
￼                "(has_repoversion_repo_model_or_obj_perms:container.delete_containerrepository "
￼                "or has_repoversion_repo_model_or_obj_perms:"
￼                "container.modify_content_containerrepository)",
```
or goes over 100 chars

Reply:
can we then split this into 2 rules?
```
            {
                "action": ["destroy"],
                "principal": "authenticated",
                "effect": "allow",
                "condition":
                    "has_repoversion_repo_model_or_obj_perms:container.delete_containerrepository "
            },
            {
                "action": ["destroy"],
                "principal": "authenticated",
                "effect": "allow",
                "condition":
                    " has_repoversion_repo_model_or_obj_perms:"
                        "container.modify_content_containerrepository)",
            },
```

This should be equivalent